developerportal/apps/externalcontent/models.py

```diff
@@ -201,8 +201,10 @@ class Meta:
 
     # Search config
     search_fields = BasePage.search_fields + [  # Inherit search_fields from Page
-        index.SearchField("description")
         # "title" is already specced in BasePage
+        index.SearchField("description"),
+        # Add FilterFields for things we may be filtering on (eg topics)
+        index.FilterField("slug"),
```

Comment:
This is a requirement of `wagtailsearch`, because the queryset involved has had a `filter()` performed on it